python/ray/train/examples/experiment_tracking/lightning_exp_tracking_comet.py

```diff
@@ -0,0 +1,39 @@
+from lightning_exp_tracking_model_dl import DummyModel, dataloader
+
+# __lightning_experiment_tracking_comet_start__
+import os
+import pytorch_lightning as pl
+from pytorch_lightning.loggers.comet import CometLogger
+import ray
+from ray.train import ScalingConfig
+from ray.train.torch import TorchTrainer
+
+
+def train_func(config):
+    logger = None
+    if ray.train.get_context().get_world_rank() == 0:
+        logger = CometLogger(api_key=os.environ["COMET_API_KEY"])
+
+    ptl_trainer = pl.Trainer(
+        max_epochs=5,
+        accelerator="cpu",
+        logger=logger,
+        log_every_n_steps=1,
+    )
+    model = DummyModel()
+    ptl_trainer.fit(model, train_dataloaders=dataloader)
+
+
+scaling_config = ScalingConfig(num_workers=4, use_gpu=False)
+
+assert (
+    "COMET_API_KEY" in os.environ
```

Comment:
The formatting in these examples are a bit messy. Could you fix it, or enable a scrollbar for them? I think there's a style parameter called `white-space: pre-wrap;` that needs to be turned off.